app/src/examples/AnimatedTextWidthExample.tsx

```diff
@@ -23,7 +23,7 @@ export default function AnimatedTextWidthExample() {
   });
 
   const animatedStyle = useAnimatedStyle(() => {
-    return { width: (1.4 + Math.sin(sv.value)) * 150 };
+    return { width: (1.4 + Math.sin(sv.value)) * 150, fontSize: 16 };
```

Comment:
It looks like `fontSize` is already set in `styles.text` so we can revert this change.